txjuju/tests/test___init__.py

```diff
@@ -1,12 +1,15 @@
 # Copyright 2016 Canonical Limited.  All rights reserved.
 
+import os
+import shutil
+import tempfile
 import unittest
 
 import txjuju
 from txjuju import cli
 
 
-class GetCLIClassTest(unittest.TestCase):
+class GetCLIClassTests(unittest.TestCase):
```

Comment:
I am not sure of the code style guidelines for txjuju, but we generally prefer singular "Test" standing for a "test case class" that naturally contains multiple tests. No biggie, of course, and this might also be an unwritten rule of Landscape code that makes no sense in this context.